source/MRViewer/MRSpaceMouseHandlerWindows.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#ifdef _WIN32
+#include "MRSpaceMouseHandler.h"
+
+namespace MR
+{
+
+class SpaceMouseHandlerWindows : public SpaceMouseHandler
```

Comment:
please add comments to all new structures and its public methods and fields

Reply:
fixed